doc/mgr/telemetry.rst

```diff
@@ -0,0 +1,36 @@
+Telemetry plugin
+================
+The telemetry plugin sends anonymous data about the cluster, in which it is running, back to the Ceph project.
+
+The data being send back to the project does not contain any sensitive data like pool names, object names, object contents or hostnames.
+
+It contains counters and statistics on how the cluster has been deployed, the version of Ceph, the distribition it runs on and other parameters which help the project to gain a better understanding of the way Ceph is used.
```

Comment:
might want to s/it/of the hosts/. as the second "it" should be indicating the same object as the first "It", but it is not.